cnd/src/db/tables.rs

```diff
@@ -189,6 +189,46 @@ impl InsertableHalight {
     }
 }
 
+#[derive(Associations, Clone, Copy, Debug, Identifiable, Queryable, PartialEq)]
+#[belongs_to(Swap)]
+#[table_name = "hbits"]
+pub struct Hbit {
+    id: i32,
+    swap_id: i32,
+    pub amount: Text<Satoshis>, // FIXME: Do we want to store this as Satoshis?
```

Comment:
Would the alternative be to store it in Bitcoin? 
I think storing Satoshi makes sense to not have to convert Satoshi to Bitcoin and not use floating point numbers.